### ISSUE TITLE
Empty header within Footer

### ISSUE THREAD

**Issue by mvogelgesang** [GSA/uswds-sf-lightning-community#54]
While sites are supposed to display a Contact Us header and associated email and or phone, if left blank, it results in an empty header which fails 508 accessibility tests. Update this section so that it conditionally displays.

https://github.com/GSA/uswds-sf-lightning-community/blob/260df2f5ed75b8fef155f4f2794e03044d0c93f6/src/aura/uswdsTheme/uswdsTheme.cmp#L227-L229

**Testing Instructions**

- deploy code
- Change community to USWDS Theme
- Before updating any theme settings, publish community and navigate to public page.
- Inspect element
```html
<div class="usa-footer__contact-links mobile-lg:grid-col-12">
  <div class="grid-row grid-gap">
    <div class="mobile-lg:grid-col-6 margin-auto zero-padding-bottom footer-float-left" >
    </div>
    <div class="mobile-lg:grid-col-6">
      <!-- there should not be any content inside this div -->
    </div>
  </div>
</div>
```
- Go back to experience builder, enter "Contact Us" in the "Contact Center Name" field. Tab out of field
- "Contact Us" should appear in the footer
- Publish
- Inspect element on public-facing site, content should appear in the second column within `usa-footer__contact-links`
- Go back to experience builder, enter values in one or more fields 
  - Contact Center Email
  - Contact Center Phone
  - Contact Center URL & Contact Center Link Text
- Values should appear in footer
- Publish and observe `<address>` markup within the second column of the footer
